cmake/FindLIBNUMA.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+message(STATUS "Checking for module 'libnuma' w/o PkgConfig")
+
+find_library(LIBNUMA_LIBRARY NAMES libnuma numa)
+set(LIBNUMA_LIBRARIES ${LIBNUMA_LIBRARY})
```

Comment:
do we need to introduce this new variable? LIBNUMA_LIBRARY isn't enough?

Reply:
So what do you suggest? It works as is now.